source/unified-test-format/tests/valid-pass/matches-lte-operator.yml

```diff
@@ -0,0 +1,40 @@
+description: matches-lte-operator
+
+schemaVersion: '1.5'
+
+createEntities:
+  - client:
+      id: &client0 client0
+      observeEvents: [ commandStartedEvent ]
+  - database:
+      id: &database0 database0
+      client: *client0
+      databaseName: &database0Name database0Name
+  - collection:
+      id: &collection0 collection0
+      database: *database0
+      collectionName: &collection0Name coll0
+
+initialData:
+  - collectionName: *collection0Name
+    databaseName: *database0Name
+    documents: []
+
+tests:
+  - description: special lte matching operator
+    operations:
+      - name: insertOne
+        object: *collection0
+        arguments:
+          document: { _id : 1, y: 1 }
+    expectEvents:
+      - client: *client0
+        events:
+          - commandStartedEvent:
+              command:
+                insert: *collection0Name
+                documents:
+                  # We can make exact assertions here but we use the $$lte operator to ensure drivers support it.
```

Comment:
I'm not sure if this was a leftover comment, but if not: what is "exact assertions" referring to? Are you implying that this is incomplete because we cannot assert a case where `$$lte`` fails to match? Presumably we could do that with a `valid-fail` test, although we wouldn't be able to confirm the exact cause of failure.

Reply:
The comment is intended to say that we could perform an exact comparison with the document `{_id: 1, y: 1}` but we use `$$lte` instead to test support.

Reply:
Oh, that makes more sense.